recipes/nnlojet/recipe.yaml

```
@@ -0,0 +1,102 @@
+context:
+  name: "nnlojet"
+  version: "1.0.0"
+
+package:
+  name: ${{ name }}
+  version: ${{ version }}
+
+source:
+  url: https://nnlojet.hepforge.org/nnlojet-v${{ version }}.tar.gz
+  sha256: b02013922387cf37745487d23b271eaae257c8af76bb3cca6bbf885dd8139e74
+
+build:
+  skip:
+    - win
+  number: 0
+  script:
+    env:
+      BASIC_ARGS: |
+        -DCMAKE_BUILD_TYPE=Release
+
+    content:
+      # Get an updated config.sub and config.guess
+      - cmake "${BASIC_ARGS}" -DCMAKE_INSTALL_PREFIX="${PREFIX}" -S . -B build
```

```suggestion
      - cmake "${BASIC_ARGS}" ${CMAKE_ARGS} -DCMAKE_INSTALL_PREFIX="${PREFIX}" -S . -B build
```

Can try Wolf's suggestion from Zulip: [#general > Validation C/Fortran compatibility with CMAKE_ARGS @ 💬](https://conda-forge.zulipchat.com/#narrow/channel/457337-general/topic/Validation.20C.2FFortran.20compatibility.20with.20CMAKE_ARGS/near/524923875)